DOCKER.md

```diff
@@ -0,0 +1,63 @@
+# Cross compiling with `Docker`
+The manual process kind of pollutes your workstation by setting lots of environment variables and writing to various config files. Thus it can be difficult to remove or upgrade these files and settings or even repeat that process for different versions of rust on the same machine.
```

Comment:
Sorry, I read it again and I know what you mean. It was not my intention to comment on the specific implemented process. Just changed this part of the manual accordingly in the latest commit.